README.Rmd

```diff
@@ -132,7 +132,7 @@ adsl %>%
   xportr_label(var_spec, "ADSL", verbose = "warn") %>%
   xportr_order(var_spec, "ADSL", verbose = "warn") %>%
   xportr_format(var_spec, "ADSL") %>%
-  xportr_write("adsl.xpt", label = "Subject-Level Analysis Dataset")
+  xportr_write("adsl.xpt")
```

Comment:
If we don't use the xportr_metadata() how does xportr_Write() get the dataset label in this method?

Reply:
do we need xportr_df_label() here???

Reply:
Good catch Ben! Yeah, in this case, the df label would be NULL we have to use the `xportr_df_label`. I will make this change wherever we do this.